cola/widgets/main.py

```diff
@@ -76,6 +76,10 @@ def __init__(self, model, parent=None, settings=None):
         # Runs asynchronous tasks
         self.runtask = qtutils.RunTask()
 
+        from cola.widgets.toolbar import create_toolbar
+        self.toolbar = self.addToolBar('Main Toolbar')
+        create_toolbar(self.toolbar)
```

Comment:
Instead of a `Main Toolbar` it might be simplest to make the code reusable and then have `Top Toolbar`, `Bottom Toolbar`, `Left Toolbar`, and `Right Toolbar`, and reuse as much as possible.

They would all default to hidden.